src/app/guidelines/page.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { Box, Typography, Button, Container } from "@mui/material";
+
+const GuidelinesScreen: React.FC = () => {
+	return (
+		<Box
```

Comment:
The containers could use some padding to create a more breathable layout and improve the overall spacing relative to the background.